eng/Versions.props

```diff
@@ -94,33 +94,35 @@
     <MicrosoftNETFrameworkReferenceAssemblies>1.0.0-alpha-5</MicrosoftNETFrameworkReferenceAssemblies>
     <MicrosoftNetRoslynDiagnosticsPackageVersion>2.6.3</MicrosoftNetRoslynDiagnosticsPackageVersion>
     <MicrosoftVisualStudioComponentModelHostPackageVersion>16.0.467</MicrosoftVisualStudioComponentModelHostPackageVersion>
-    <MicrosoftVisualStudioImageCatalogPackageVersion>16.0.28727</MicrosoftVisualStudioImageCatalogPackageVersion>
+    <MicrosoftVisualStudioImageCatalogPackageVersion>16.4.29519.181</MicrosoftVisualStudioImageCatalogPackageVersion>
```

Comment:
Where will 'nimullen/asyncpackage' eventually merge in? If it's downlevel, I worry about these being breaking changes.

Reply:
asyncpackage already merged, I need to rebase this branch to be against master 😄. 

Out of curiosity what type of breaking changes do you feel will come from this? I ask because these package versions should technically only impact our VisualStudio bits.

Reply:
Compat w/ different VS versions and perhaps behaviour changes since you're upping the major version of a few dependencies

Reply:
AH, so the versions in practice actually aren't different at all. These versions are ones that are shipped with VS so they get binding redirected to the newer versions anyhow 😄